src/httpServer.js

```diff
@@ -0,0 +1,116 @@
+const http = require('http')
+const logger = require('./logger')
+const { nanoid } = require('nanoid')
+const { AsyncLocalStorage } = require('async_hooks') 
```

Comment:
why not to add async local storage to express server?

Reply:
In Express app there is already "express-pino-logger" which does the work (log request context, etc) in more effective and convenient way.

Reply:
I see